toolsay/toolsay.csproj

```diff
@@ -1,54 +1,56 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
+    <OutputType Condition="'$(PackAsTool)' == 'true'" >Exe</OutputType>
     <TargetFramework>net10.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
We can't rely on globbing because we don't want Program.cs in the library part of the package.